File: jetbrains/src/main/java/com/sourcegraph/cody/agent/CodyAgentException.java
package com.sourcegraph.cody.agent;

public class CodyAgentException extends Exception {
  public CodyAgentException(String message) {
    super(message);
  }

  public CodyAgentException(String message, Exception e) {
    super(message, e);
  }

  @Override
  public Throwable fillInStackTrace() {
    // don't fill in stack trace
    return this;
  }
}

File: jetbrains/src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentException.kt
package com.sourcegraph.cody.agent

class CodyAgentException : Exception {
  constructor(message: String?) : super(message)

  constructor(message: String?, e: Exception?) : super(message, e)

  override fun fillInStackTrace(): Throwable {
    // don't fill in stack trace
    return this
  }
}
